### PR TITLE
Update help text of license id formats

### PR DESCRIPTION
**Pull request by mabuaisha**

### CODE REVIEW

**Comment by mabuaisha** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;JdwCXF2p&#x2F;653-1-register-form-wizard-registration-for-license-id-format-placeholder-not-fully-displayed-cycle-12-cycle-13"> (1) Register - Form Wizard Registration For License ID Format Placeholder Not Fully Displayed [Cycle 12] [Cycle 13]</a>

File: registration/forms.py [hunk 1]
```diff
@@ -247,25 +247,32 @@ def register_form_generator(conf):
...
+                elif field_name == 'email':
+                    d['max_length'] = max_length
+                    d['help_text'] = help_text
+                    field_class = django.forms.EmailField
+                elif field_name == 'license_id'\
+                        and 'license_id_formats' in conf:
+                    d['max_length'] = max_length
+                    license_id_formats = '<p class=\'hint-license-id-format\'>' \
+                                         'Valid Formats: ' \
+                                         + ', '.join(map(str, conf['license_id_formats'])) + '</p>'
```
Comment (danaspiegel) [inline registration/forms.py]: this should map ``unicode`` not ``str``

File: registration/forms.py [hunk 2]
```diff
@@ -247,25 +247,32 @@ def register_form_generator(conf):
...
+                    help_text = '<p>' + unicode(help_text) + '</p>'
+                    license_id_formats += help_text
```
Comment (danaspiegel) [inline registration/forms.py]: @mabuaisha this shouldn't use ``+=``. It should use string templates ("{}".format)

File: registration/forms.py [hunk 3]
```diff
@@ -247,25 +247,32 @@ def register_form_generator(conf):
...
+                    d['max_length'] = max_length
+                    d['help_text'] = help_text
+                    field_class = django.forms.EmailField
+                elif field_name == 'license_id'\
+                        and 'license_id_formats' in conf:
+                    d['max_length'] = max_length
+                    license_id_formats = '<p class=\'hint-license-id-format\'>' \
+                                         'Valid Formats: ' \
+                                         + ', '.join(map(str, conf['license_id_formats'])) + '</p>'
+                    help_text = '<p>' + unicode(help_text) + '</p>'
```
Comment (danaspiegel) [inline registration/forms.py]: @mabuaisha this shouldn't use ``+``. It should use string templates ("{}".format)

File: static/css/base.css [hunk 4]
```diff
@@ -761,3 +761,8 @@ body.upenn-start {
     font-size: larger;
     color: #ffffff;
 }
+
+.help-block .hint-license-id-format {
+    font-size: medium;
+    font-weight: bolder;
```
Comment (danaspiegel) [inline static/css/base.css]: not bold... italics.

File: registration/forms.py [hunk 5]
```diff
@@ -247,25 +247,32 @@ def register_form_generator(conf):
...
+                    field_class = django.forms.ChoiceField
+                elif field_name == 'email':
+                    d['max_length'] = max_length
+                    d['help_text'] = help_text
+                    field_class = django.forms.EmailField
+                elif field_name == 'license_id'\
+                        and 'license_id_formats' in conf:
+                    d['max_length'] = max_length
+                    license_id_formats = '<p class=\'hint-license-id-format\'>' \
+                                         'Valid Formats: ' \
```
Comment (danaspiegel) [inline registration/forms.py]: "Valid License IDs look like any of "

**Pull request merged**